core/app/models/spree/ability.rb

```diff
@@ -28,9 +28,8 @@ def initialize(current_user)
       @user = current_user || Spree.user_class.new
 
       alias_actions
-      grant_default_permissions
-      register_extension_abilities
       activate_permission_sets
+      register_extension_abilities
```

Comment:
this is great also that extension abilities get evaluated after the permission sets, makes it far easier to add behavior